contrib/cirrus/win-installer-main.ps1

```diff
@@ -12,6 +12,7 @@ if ($Env:CI -eq "true") {
     $RELEASE_DIR = "$PSScriptRoot\..\..\contrib\win-installer\current"
     if ($null -eq $ENV:CONTAINERS_MACHINE_PROVIDER) { $ENV:CONTAINERS_MACHINE_PROVIDER = 'wsl' }
 }
+$ENV:LATEST_GH_RELEASE_ID = "199677288" # v5.4.1
```

Comment:
shouldn't this be 5.4.0?

Reply:
Oops! Yes the version in the comment was wrong but the release ID was the right one. I have fixed it.